sqlcl/datamove-schema/readme.md

```diff
@@ -0,0 +1,23 @@
+# datamove-schema scripts
+
+A set of scripts to datapump a schema from a local database to an ADW cloud database.
+
+*Only tested with Oracle XE 18 (18.4.0.0.0) (local) and Oracle Database 19c (19.5.0.0.0) (ADW)* 

```

Comment:
Could we give this statement a bit more positive spin, e..g "Tested on Oracle XE 18c and Oracle Autonomous Data Warehouse" or similar? :)

Reply:
Whoops, one more change I guess